fastembed/text/sparse.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Dict, List, Tuple, Type
+
+import numpy as np
+
+from fastembed.text.onnx_embedding import (
+    EmbeddingWorker,
+    OnnxTextEmbedding,
+    OnnxTextEmbeddingWorker,
+)
+
+supported_splade_models = [
+    {
+        "model": "prithvida/SPLADE_PP_en_v1",
+        "dim": 30522,  # vocab size
+        "description": "Independent Implementation of SPLADE++ Model for English",
+        "size_in_GB": 0.532,
+        "sources": {
+            "hf": "nirantk/SPLADE_PP_en_v1",
```

Comment:
This model won't be added to the Qdrant HF account?

Reply:
yeah, it should be under our account